joss-paper/paper.md

```diff
@@ -0,0 +1,123 @@
+---
+title: '21cmFAST v3: A fully Python-integrated simulation code to generate 3D 21cm fluctuation fields through cosmic history.'
+tags:
+  - Python
+  - astronomy
+  - cosmology
+  - simulation
+authors:
+  - name: Steven G. Murray
+    orcid: 0000-0003-3059-3823
+    affiliation: 1
+  - name: Bradley Greig
+    orcid: xxxxxx
+    affiliation: 2
+  - name: Andrei Mesinger
+    orcid: xxxxxx
+    affiliation: 3
+affiliations:
+ - name: School of Earth and Space Exploration, Arizona State University, Phoenix, USA
+   index: 1
+date: 29 Feb 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+The field of 21cm cosmology -- in which the hyperfine spectral line of neutral hydrogen
+is mapped over large swathes of the Universe's history -- has developed radically over
+the last decade.
+Notwithstanding the overwhelming technical challenges associated with the observation
+and eventual detection of this signal, the promise of the field is to revolutionise our
+knowledge of the first light sources: a worthy prize indeed.
+In order to interpret the eventual observational data, a range of physical models have
+been developed -- from simple analytic models of the global history of reionization,
+through to fully hydrodynamical simulations of the 3D evolution of the brightness
+temperature of the spectral line.
+Between these extremes lies an especially versatile middle-ground: fast semi-analytic
+models that approximate the full 3D evolution of the field.
+These have the advantage of being comparable to the full first-principles
+hydrodynamic simulations, but significantly quicker to run; so much so that they can
+be used to produce thousands of realisations in order to explore the very wide
+parameter space that still remains consistent with the data.
+
+
+Amongst practioners in the field of 21cm cosmology, the `21cmFAST` program has become
+the *de facto* standard for such semi-analytic simulators. `21cmFAST` [@mesinger2010]
+is a high-performance C code that uses the excursion set formalism [@furlanetto2004] to
+identify regions of ionized hydrogen atop a cosmological density field evolved using
+first- or second-order perturbation theory.
```

Comment:
Lagrangian perturbation theory (you can cite e.g. zel'dovich 1970 here and Scocciomarro and Sheth 2002

The code also tracks the thermal and ionization state of the intergalactic medium, and computes X-ray, soft UV and ionizing UV cosmic radiation fields based on parametrized galaxy models.

Reply:
Thanks, added it in.